### PR TITLE
Update `LogInv` in the TLA+ spec

### PR DESCRIPTION
**Pull request by heidihoward**
This PR strengthens and simplifies the `LogInv` invariant. This invariant is the key safety invariant we check (alongside `CommittedLogNeverChangesProp`) so I've given this small change a dedicated PR.

### CODE REVIEW

File: tla/ccfraft.tla [hunk 1]
```diff
@@ -1141,9 +1129,14 @@ Spec == Init /\ [][Next]_vars
 \* Correctness invariants
 \* These invariants should be true for all possible states
 
-\* Committed log entries should not conflict
+\* Committed log entries should never conflict between servers
+\* This is a key safety invariant and should always be checked
+\* LogInv checks for safety violations across space
```
Comment (lemmy) [inline tla/ccfraft.tla:1134]: Perhaps, this statement can be better illustrated by stating it as a theorem:

```tla
THEOREM Spec => []LogInv

THEROEM Spec => CommittedLogNeverChangesProp
```

File: tla/ccfraft.tla [hunk 2]
```diff
@@ -1284,6 +1277,8 @@ MonotonicCommitIndexProp ==
     [][\A i \in Servers :
         commitIndex[i]' >= commitIndex[i]]_vars
```
Comment (lemmy) [inline tla/ccfraft.tla:1278]: Add a note that `CommittedLogNeverChangesProp` is a much stronger variant of `MonotonicCommitIndexProp`?

File: tla/ccfraft.tla (hunk 1 above)
Comment (heidihoward) [reply tla/ccfraft.tla]: Done in cad2f39a7d710959f76bef958c62dce182020c23.

**Comment by heidihoward** [comment]
This PR is ready to merge

**Review by achamayou — APPROVED** [review]

**Pull request merged**